src/charts/gv-chart-gauge.js

```diff
@@ -15,16 +15,25 @@
  */
 import { LitElement } from 'lit-element';
 import { ChartElement } from '../mixins/chart-element';
+import ChartModuleMore from 'highcharts/highcharts-more';
+import Highcharts from 'highcharts';
+import HCSolidGauge from 'highcharts/modules/solid-gauge';
 
 /**
  * Gauge chart component
  *
- * @attr {Array} max - The series to display on the pie chart.
+ * @attr {Array} max - The maximum value of the gauge.
```

Comment:
Still not an `Array ` 😛

Reply:
That would not happen if you used suggestions :wink: